docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/identifying-policy-in-use.adoc

```diff
@@ -0,0 +1,24 @@
+= Identifying Policy in-use
+
+In Konflux, there are two places where Enterprise Contract Policies are used, integration tests and
```

Comment:
Is it more accurate to say that there are two places that EC is used by default? I guess there are not really other places where EC can be easily used other than these two.

Reply:
Add "by default" as I think that is more accurate.